graylog2-web-interface/src/components/authentication/activeDirectory/BackendCreate.jsx

```diff
@@ -0,0 +1,66 @@
+// @flow strict
+import * as React from 'react';
+
+import { DocumentTitle } from 'components/common';
+import BackendWizard from 'components/authentication/BackendWizard';
+import AuthenticationDomain from 'domainActions/authentication/AuthenticationDomain';
+
+import WizardPageHeader from './WizardPageHeader';
+
+export const HELP = {
```

Comment:
This is a bit awkward, that you define it here but never use it in this component. Perhaps move it to its own file. What do you think?

Reply:
I missed thank one, thanks!